dev-python/sphinx-press-theme/sphinx-press-theme-0.8.0.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYPI_NO_NORMALIZE=1
+PYPI_PN="sphinx_press_theme"
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="A Sphinx-doc theme based on Vuepress"
+HOMEPAGE="https://github.com/schettino72/sphinx_press_theme"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
+
+RDEPEND="dev-python/sphinx[${PYTHON_USEDEP}]"
+
+RESTRICT="mirror"
```

Comment:
Why?

Reply:
Also, no RESTRICT="test"? (In the commit message, you mention tests are broken, so add a comment explaining why/how?)

Reply:
RESTRICT="mirror" should be gone. Forgot to remove that. Will update.

The tests for media-libs/opencolorio are broken. This has none as far as I know.

Reply:
This is for `media-libs/opencolorio[doc]`, not for `test` fumbled this because I kept reading testressources.